src/utils/components/FirmwareBootloaderModal/utils/utils.ts

```diff
@@ -1,14 +1,20 @@
 import { TFunction } from 'i18next';
+import produce from 'immer';
 
 import { V1VirtualMachine } from '@kubevirt-ui/kubevirt-api/kubevirt';
+import { ensurePath, isEmpty } from '@kubevirt-utils/utils/utils';
 
 import { BootloaderOptionValue } from './constants';
 
+export const isObjectEmpty = (obj: object): boolean => isEmpty(obj) && obj !== undefined;
```

Comment:
What about case null? Is possible? i think isEmpty(obj) && obj can work also?

Reply:
In the place I am using this function, null isn't possible. But if someone decided to use this somewhere else, probably `isEmpty(obj) && obj` is better to use.

Reply:
Can you explain to me why just isEmpty(obj) is not enough? it should return true if it's either one of these:
`null / undefind / [] / {} / ''`

Reply:
Because we don't want to receive true if the object is null or undefined.

Reply:
so `efi: {}` and `efi: { secureBoot: true }` is equivalent right? and if `efi` does not exist we want bios?

Reply:
Yes, they both refer to secure boot enabled displayed as "UEFI (secure)" option in the UI. And yes, if no efi, then show "BIOS". This was the original implementation at least. I was not dealing with this third option too much, the main problem was the secure boot option.